### PR TITLE
Set default channel to 4.6

### PR DESCRIPTION
**Pull request by warrenvw**
Since 1.11 only targets 4.6, we can now set the default channel to 4.6. Shawn concurs.

### CODE REVIEW

**Comment by mgencur** [comment]
Wow! That's great! :-)
Anyway, this change needs to be applied in this way:
* modify the version at https://github.com/openshift-knative/serverless-operator/blob/master/olm-catalog/serverless-operator/project.yaml#L9 and https://github.com/openshift-knative/serverless-operator/blob/master/olm-catalog/serverless-operator/project.yaml#L19
* re-generate the target files by `make release-files`
This will do the exact changes on the files you have now

File: olm-catalog/serverless-operator/Dockerfile [hunk 1]
```diff
@@ -7,8 +7,8 @@ LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=serverless-operator
-LABEL operators.operatorframework.io.bundle.channel.default.v1="4.3"
-LABEL operators.operatorframework.io.bundle.channels.v1="4.5,4.6"
+LABEL operators.operatorframework.io.bundle.channel.default.v1="4.6"
+LABEL operators.operatorframework.io.bundle.channels.v1="4.6"
```
Comment (matzew) [inline olm-catalog/serverless-operator/Dockerfile:11]: glad to see this defaulting

**Comment by warrenvw** [comment]
I started on the fix as soon as I saw the failed generate test. Thanks for the review.. :)

File: olm-catalog/serverless-operator/Dockerfile (hunk 1 above)
Comment (warrenvw) [reply olm-catalog/serverless-operator/Dockerfile:11]: Yeah, me too. And I'm sure @cardil as well. :)

File: templates/main.Dockerfile [hunk 2]
```diff
@@ -20,4 +20,4 @@ LABEL \
       io.k8s.display-name="Red Hat OpenShift Serverless Bundle" \
       com.redhat.openshift.versions="__OCP_TARGET_VLIST__" \
       com.redhat.delivery.operator.bundle=true \
-      com.redhat.delivery.backport=true
+      com.redhat.delivery.backport=false
```
Comment (mgencur) [inline templates/main.Dockerfile:23]: Just curious. What does this mean?

**Comment by warrenvw** [comment]
More importantly, looks to me like we need to run CI on 4.6, not 4.5...

File: templates/main.Dockerfile (hunk 2 above)
Comment (warrenvw) [reply templates/main.Dockerfile:23]: We do not need to backport the bundle to the appregistry format since 1.11 is targeting 4.6.

**Comment by mgencur** [comment]
Yeah. We should probably switch it. However, 4.6 is not even GA yet. Also, we have a nightly job on 4.6 (with slack notifications to serverless-dev) which currently fails. I think it's due to https://bugzilla.redhat.com/show_bug.cgi?id=1883589 which is being investigated on OpenShift side.
I can try to apply workarounds from the BZ and see if tests on 4.6 start passing. 
Anyway, it's possible right now to trigger tests on 4.6 manually on each PR against master branch. E.g. like this: https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master-presubmits.yaml#L472

File: templates/main.Dockerfile (hunk 2 above)
Comment (cardil) [reply templates/main.Dockerfile:23]: I thought we will try to use separate bundles to our advantage. What I mean is to deploy specific versions of Serverless to specific OCP 4.6+ versions, and having channel called `stable` in all of them. That should be possible with the new bundle format, right?

https://issues.redhat.com/browse/SRVCOM-886

Comment (mgencur) [reply templates/main.Dockerfile:23]: @cardil this is not the right place to resolve this. I don't remember discussions about changing the channel strategy in this way from our program calls. Maybe you can bring it up there.

Comment (cardil) [reply templates/main.Dockerfile:23]: Sure. I think the same.

**Comment by markusthoemmes** [comment]
/retest

Lovely infra.

File: templates/main.Dockerfile (hunk 2 above)
Comment (warrenvw) [reply templates/main.Dockerfile:23]: Per https://issues.redhat.com/browse/RFE-723 and discussions I've had with Shawn, we'll be able to use a `stable` channel beginning with OCP 4.7.

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Pull request merged**